packages/scalars/src/scalars/index.ts

```diff
@@ -0,0 +1,7 @@
+import { GraphQLEmailAddress } from './EmailAddress';
+
+export * from './EmailAddress';
```

Comment:
Here it would be just `export * as EmailAddress from './EmailAddress"`.

Reply:
In the future we could create a `nx generate scalar EmailAddress` that creates the boilerplate and updates this file.